.github/README.md

```
@@ -28,38 +26,38 @@
 
-- [About The Project](#-about-the-project)
-- [The Problem](#-the-problem)
-- [Screenshots](#-screenshots)
-- [Architecture](#-architecture)
-- [Tech Stack](#-tech-stack)
+- [About The Project](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/tree/Bug-Resolve/main-README#about-the-project)
+- [The Problem](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/tree/Bug-Resolve/main-README#the-problem)
+- [Screenshots](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/tree/Bug-Resolve/main-README#screenshots)
+- [Architecture](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/tree/Bug-Resolve/main-README#architecture)
+- [Tech Stack](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/tree/Bug-Resolve/main-README#tech-stack)
 - [Features](#-features)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)